src/dotnet/APIView/APIViewWeb/Models/RenderedCodeFile.cs

```diff
@@ -50,16 +50,16 @@ public CodeLine[] RenderReadOnly(bool showDocumentation)
             return _renderedReadOnly;
         }
 
-        internal CodeLine[] RenderText(bool showDocumentation)
+        internal CodeLine[] RenderText(bool showDocumentation, bool skipDiff = false)
         {
             if (showDocumentation)
             {
-                return CodeFileRenderer.Instance.Render(CodeFile, showDocumentation: true, enableSkipDiff: true);
+                return CodeFileRenderer.Instance.Render(CodeFile, showDocumentation: showDocumentation, enableSkipDiff: skipDiff);
             }
 
             if (_renderedText == null)
             {
-                _renderedText = CodeFileRenderer.Instance.Render(CodeFile, enableSkipDiff: true);
+                _renderedText = CodeFileRenderer.Instance.Render(CodeFile, enableSkipDiff: skipDiff);
```

Comment:
You would end up caching the value the first time you render. It might be with or without skipDiff

Reply:
I think I will use the same approach as documentation to always render if those properties are set. Thanks. Currently this flag is set only for diff comparison for auto reviews to decide whether to generate a new revision or not.

Reply:
done